setup/ci/jenkins.md

```diff
@@ -121,29 +129,37 @@ In the UI, go to `"Manage Jenkins"` >> `"Configure System"` and set your git `us
 At this point, you should be able to manually run the script job in Jenkins
 (with parameters) and see it succeed.
 
-### Configuring Spinnaker
-Enable Igor.
+### Configure Spinnaker
 
-In spinnaker-local.yml, set:
+1. Make sure Igor is enabled.
 
-```
-jenkins.enabled = true
-jenkins.masters[0].name = <jenkins_name>
-jenkins.masters[0].address = http://<jenkins_host>/jenkins Note that "/jenkins" might not be the base path, it depends on how Jenkins is configured.
-jenkins.masters[0].username = <username>
-jenkins.masters[0].password = <user_api_token>
-```
+   Because you [enabled Jenkins](#add-your-jenkins-master), Igor should be
+   enabled automatically by Spinnaker. You can confirm this by checking your
+   `~/.hal/default/service-settings/`
 
-In orca-local.yml, add:
+   If it's not enabled, you can override it to enable it in
+   `~/.hal/default/service-settings/igor.yml`:
 
-```
-script:
-  master: <jenkins_name> # name of Jenkins master in Spinnaker
-  job: <JOB_NAME> # from Jenkins job configuration
-```
+   `enabled:true`
+
+1. In spinnaker-local.yml, set:
```

Comment:
Is this needed _in addition to_ the `hal config ci jenkins master add` call that was made earlier? (Same question for the `orca-local.yml` below.)

Reply:
Yeah, that's a good point. That might be vestigial, from before halyard. I'll have a new PR ready in the morning. 
Thanks!